AdyenCard/Utilities/BinLookup/BinLookupResponse.swift

```diff
@@ -31,21 +31,26 @@ internal struct BinLookupResponse: Response {
     /// :nodoc:
     internal let enableLuhnCheck: Bool?
 
+    /// :nodoc:
+    internal let cvcPolicy: CardBrand.CVCPolicy?
```

Comment:
cvcPolicy is not one for all brands, its can be different for each detected brand.